docs/source/workflow.rst

```diff
@@ -193,3 +193,51 @@ monitoring checks for free:
   instances are deployed on a cluster. If the service is registered in Smartstack
   it will look in Smartstack to count the available instances. Otherwise it
   counts the number of healthy tasks in Mesos.
+
+
+The PaaSTA command line
+------------------------
+
+The PaaSTA command line interface, ``paasta``, gives users of PaaSTA the
+ability to inspect the state of services, as well as stop and start existing
+services. See the man pages for a description and detail of options for any
+individual paasta command.  Some of the most frequently used commands are
+listed below:
+
+   * ``paasta start`` - sets the desired state of the service instance to
```

Comment:
I can't figure out how to do the sphinx magic - I got to the last hurdle, but can't get the ``:py:meth:help`` working.

>  I will ask that you copy these docs into the help text of the commands themselves so they show up with --help

Done for emergency-start and emergency-stop.

It's difficult for start and stop because we auto generate the contents of the description field. See [here](https://github.com/Yelp/paasta/blob/master/paasta_tools/cli/cmds/start_stop_restart.py#L33-L47)